packages/web-components/src/components/masthead/__stories__/masthead.stories.ts

```diff
@@ -286,6 +287,28 @@ withAlternateLogoAndTooltip.story = {
   },
 };
 
+export const WithCloudVariant = ({ parameters }) => {
+  const { selectedMenuItem, userStatus, searchPlaceholder, hasProfile, hasSearch } = parameters?.props?.MastheadComposite ?? {};
+  return html`
```

Comment:
@huntermacd Let's use the `useMock` variable so that we're keeping to the same method for Percy's visual testing for cloud-masthead components as well.

```suggestion
  const { useMock } = parameters?.props?.Other ?? {};
  return html`
    <style>
      ${styles}
    </style>
    ${useMock
      ? html`
          <dds-masthead-composite
            platform="${ifNonNull(platformData.name)}"
            platform-url="${ifNonNull(platformData.url)}"
            selected-menu-item="${ifNonNull(selectedMenuItem)}"
            user-status="${ifNonNull(userStatus)}"
            searchPlaceholder="${ifNonNull(searchPlaceholder)}"
            .authenticatedProfileItems="${ifNonNull(authenticatedProfileItems)}"
            .navLinks="${navLinks}"
            ?has-profile="${hasProfile}"
            ?has-search="${hasSearch}"
            .unauthenticatedProfileItems="${ifNonNull(unauthenticatedProfileItems)}"
          ></dds-masthead-composite>
        `
      : html`
          <dds-masthead-container
            platform="Cloud"
            platform-url="${ifNonNull(platformData.url)}"
            selected-menu-item="${ifNonNull(selectedMenuItem)}"
            user-status="${ifNonNull(userStatus)}"
            searchPlaceholder="${ifNonNull(searchPlaceholder)}"
            .navLinks="${navLinks}"
            ?has-profile="${hasProfile}"
            ?has-search="${hasSearch}"
            data-endpoint="/common/carbon-for-ibm-dotcom/translations/cloud-masthead/jsononly"
          ></dds-masthead-container>
        `}
  `;
```